### PR TITLE
doc/rados/operations/devices: document device failure prediction

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

**Comment by liewegas** [comment]
@hsiang41

**Comment by hsiang41** [comment]
It's good.

**Comment by badone** [comment]
jenkins render docs please

File: doc/rados/operations/devices.rst [hunk 1]
```diff
@@ -75,7 +75,42 @@ for a specific timestamp) with::
...
+can see the life expectancy of all devices in output from::
+
+  ceph device ls
+
+You can also query the metadata for a specific device with::
+
+  ceph device info <devid>
+
+You can explicitly force predict a device's life expectancy with::
+
```
Comment (badone) [inline doc/rados/operations/devices.rst]: Did you mean "You can explicitly force prediction of" maybe?

**Pull request merged**